honeybee_energy/properties/room.py

```diff
@@ -279,6 +286,200 @@ def is_conditioned(self):
+
+        This parameter is used to derive air flow for the four cracks around the
+        perimeter of a closed window or door: one along the bottom, one along the top,
+        and one on each side. Since this value is derived from the infiltration flow
+        rate per exterior area, which represents an average over many types of air
+        leakage rates, this value is not intended to be representative of actual opening
+        edges flow coefficients. Note that, unlike the surface area flow_coefficient,
+        this coefficient is normalized per unit length, which is the required input unit
+        the EnergyPlus AirflowNetwork, whereas the flow coefficient for surface cracks is
+        not normalized. The normalized perimeter air mass flow coefficient is derived
+        from its infiltration flow rate using the following formula:
+
+            Qv * d * A = Cql * L * dP^n
+
+            where:
+                Cql: Air mass flow coefficient per unit length at 1 Pa [kg/m/s/P^n]
+                Qv: Volumetric air flow rate per length [m3/s/m]
+                d: Air density [kg/m3]
+                A: Surface area of opening [m2]
+                L: Surface perimeter of opening [m]
+                dP: Change in pressure across building envelope [Pa]
+                n: Air mass flow exponent [-]
+
+        Since Qv * d / dP^n equals Cqa the normalized area flow coefficient, this can
+        be simplified and rearranged to solve for Cql with:
+
+            Cql = Cqa * A / L
+
+        Args:
+            norm_area_flow_coefficient: Air mass flow coefficient per unit meter at
+                1 Pa [kg/m2/s/P^n]
+            face_area: A numerical value for the total exterior area in m2.
+            face_perimeter: A numerical value for the total exterior perimeter in meters.
+
+        Returns:
+            Air mass flow coefficient per unit length at 1 Pa [kg/m/s/P^n]
+        """
+        cqa = norm_area_flow_coefficient
+        a = face_area
+        ln = face_perimeter
+
+        return cqa * a / ln
+
+    def exterior_afn_from_infiltration_load(self, exterior_face_groups,
+                                            air_density=1.2041):
+        """Calculate exterior AirflowNetwork parameters from the room infiltration rate.
+
+        This will compute air leakage parameters for exterior cracks and opening edges
+        that produce a total air flow rate equivalent to the room infiltration rate, at
+        an envelope pressure difference of 4 Pa. However, the individual flow air
+        leakage parameters are not meant to be representative of real values, since the
+        infiltration flow rate is an average of the actual, variable surface flow
+        dynamics.
+
+        VentilationOpening objects will be added to Aperture and Door objects if not
+        already defined. If already defined, only the parameters defining leakage when
```

Comment:
It feels like you are removing a huge degree of freedom here if you are forcing the user to accept that all apertures suddenly become operable when this method is called.  I realize you may still need the VentilationOpening object even when the window doesn't open. But, in this case, can you ensure that the operable area of the VentilationOpening  is 0 so that we aren't totally changing the user's original assumptions here?